pkgs/development/python-modules/fusesoc/default.nix

```diff
@@ -0,0 +1,38 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+
+, verilog
+, verilator
+, gnumake
+, gcc
+
+, edalize
+, pyparsing
+, pyyaml
+, simplesat
+, ipyxact
+
+, setuptools-scm
+}:
+buildPythonPackage rec {
+  pname = "fusesoc";
+  version = "1.12.0";
+
+  propagatedBuildInputs = [ edalize pyparsing pyyaml simplesat ipyxact verilog verilator gnumake gcc ];
```

Comment:
Why are `gnumake` and `gcc`  in propagated build inputs?

Reply:
theyre needed at runtime